host-bmc/host_pdr_handler.cpp

```diff
@@ -162,18 +162,16 @@ HostPDRHandler::HostPDRHandler(
                     this->objPathMap[element.first] = obj;
                 }
             }
-            else if (
-                propVal ==
-                "xyz.openbmc_project.State.Host.HostState.TransitioningToOff")
-            {
-                isHostTransitioningToOff = true;
-            }
             else if (propVal ==
                      "xyz.openbmc_project.State.Host.HostState.Running")
             {
-                isHostTransitioningToOff = false;
+                isHostRunning = true;
                 isHostOff = false;
```

Comment:
can't we just use the `isHostOff` variable instead ? why to create a new variable ? , since isHostOff has inverse relation to isHostRunning, we can directly use `!isHostOff` where ever we were using `isHostRunning` ?

Reply:
looked again , it looks risky now. lets not do it.